test/README.md

```diff
@@ -1,14 +1,92 @@
+## Development Culture
+
+While tests are an important part of the development process, an even more important part is the culture that surrounds them. A few core tenets of our development culture are:
+
+- Open communication is encouraged. If you have questions, ask them! If you have ideas, share them! If you have concerns, raise them!
+- Respect is required. We are all here to work together to build something great. We are all here to learn from each other, help one another, and to have fun! A respectful and collaborative environment is required to achieve this.
+- All code should be reviewed by at least one other developer before merging into `main`, preferably by multiple developers. This is to ensure that code is readable, understandable, and secure.
+- Detailed PR descriptions are encouraged. This is to ensure that reviewers can understand the context of the PR, and to ensure that the PR description can be used as a reference in the future.
+- Developers are encouraged to ask questions, and to share ideas. This is the best way to learn, and to improve the codebase.
+- Developers are encouraged to share their knowledge with others. This is the best way to improve the codebase.
+- Highlighting risks is encouraged. If you see a potential risk that is not being addressed, raise it! If you see a potential risk that is being addressed, but you think it can be addressed better, raise it!
+- Preventing bugs in deployed contracts is a team effort. If a bug is found, it should be considered an opportunity to improve our development system based on lessons learned. A single engineer is never soley responsible for a bug; many engineers are involved in our development process (writing code, reviewing code, reviewing tests, following our established process, etc.), and a bug represents a failure of our process, not an individual.
+
+# Future Improvements
+
+The following are some ideas for future improvements to our testing implementation:
+
+- We do not currently use Hardhat fixtures to speed up tests. We should refactor our tests to use fixtures where possible.
+  - (captured in https://github.com/ArtBlocks/artblocks-contracts/issues/371)
+- We do not currently include coverage reports in our automatic CI runs. We should add this to our CI runs.
+  - (captured in https://github.com/ArtBlocks/artblocks-contracts/issues/245)
+- We do not currently implement a static analysis tool. We should add a static analysis tool, such as [slither](https://github.com/crytic/slither).
+  - (captured in https://github.com/ArtBlocks/artblocks-contracts/issues/372)
+
+Potential improvements that have benefits and drawbacks:
+
+- Forking mainnet or testnets to run tests. This would allow us to test against real-world data, but would also require us to maintain a forked node API key, and would raise the level of complexity for contributors.
+- Using git submodules to pull in (more complex) external contracts, and run tests against those contracts.
+
+# Helpful Examples
+
```

Comment:
It's just 1 method but do you think it's worth adding this to speed things up for others?
```suggestion
## Isolate Unit Tests

`.only()` is a helpful pattern to use when you're working on your tests. With this method any tests that fall outside of your isolated describe block(s) will be ignored, and this can speed up your development/testing cycle by an order of magnitude.

  describe.only("common tests", async function () {
    await AdminACLV0V1_Common("AdminACLV0");
  });
```

Reply:
I've went ahead and addressed both of your concerns in [this commit](https://github.com/ArtBlocks/artblocks-contracts/pull/373/commits/a44fb7cd8f79016cdcc7eed0fa75683adc0a77c4):
- Added @mchrupcala 's helpful tip (which is actively used by devs, even if we don't recommend in the readme)
- Added an eslint rule that causes failure if any tests use a `describe.only()` block

Thanks for the comments!